samples/shared/Mixer.h

```diff
@@ -34,35 +31,38 @@ class Mixer : public IRenderableAudio {
 public:
     void renderAudio(float *audioData, int32_t numFrames) {
 
+        int numBytes = numFrames * mChannelCount;
```

Comment:
numBytes should be "numSamples" or "numFloats"

Reply:
Oops... Thanks! I just fixed it